charts/generic-service/postgres-restore.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+set -e
+
+PRISON_API_BASE_URL=https://api-preprod.prison.service.justice.gov.uk
+
+check_http() { http --stream --check-status --ignore-stdin --timeout=600 "$@"; }
+psql_preprod() { psql -h "$DB_HOST_PREPROD" -U "$DB_USER_PREPROD" -d "$DB_NAME_PREPROD" -At -c "$@"; }
```

Comment:
Might be nice to validate all the env vars exist and are not empty up front. I can see people either missing the `namespace_secrets` or getting them wrong (some projects have differed from the default `dps-rds-instance-output`). Would be good if they saw a nice error message for that.

Reply:
yes, good idea